src/domain/synthetics/markets/utils.ts

```diff
@@ -119,11 +119,15 @@ export function getMaxReservedUsd(marketInfo: MarketInfo, isLong: boolean) {
     ? marketInfo.openInterestReserveFactorLong
```

Comment:
thinking if it makes sense to rename `getMaxReservedUsd`, it may not be clear that `maxOpenInterest` key is also checked inside 🤔